functions/go/list-setters/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+image: gcr.io/kpt-fn/list-setters
+description: List setters on resource fields.
+tags:
+  - TODO
```

Comment:
TODO: I was not sure what the tags should be

Reply:
You can go with `viewer` for now.

Reply:
Not updated ?

Reply:
updated